lib/prometheus/business_collector.rb

```diff
@@ -111,6 +117,11 @@ def collect
     @client_accounts_with_huge_policies.observe(
       client_accounts.where(id: hosts_50_plus).count(:account_number)
     )
+
+    @total_huge_policies.observe(hosts_50_plus.count.size)
```

Comment:
Observation: `hosts_50_plus` is a bit misleading. It might be smth like `policies_50plus_hosts`.